runelite-client/src/main/java/net/runelite/client/config/FlashNotification.java

```diff
@@ -32,8 +32,8 @@
 public enum FlashNotification
 {
 	DISABLED("Off"),
-	FLASH_TWO_SECONDS("Flash for 2 seconds"),
-	SOLID_TWO_SECONDS("Solid for 2 seconds"),
+	FLASH_FOR_DURATION("Flash for specified duration"),
+	SOLID_FOR_DURATION("Solid for specified duration"),
```

Comment:
Additionally this needs to be improved (though may be irrelevant with a simple off/flash/solid)

![image](https://user-images.githubusercontent.com/309739/94368755-ec632a80-00b3-11eb-9e04-d33800d7362f.png)

Reply:
Thank you for the feedback. I'll work on making these changes. It might take a while since I need to set my entire environment back up after getting a new computer but I'll let you know when I have an update.